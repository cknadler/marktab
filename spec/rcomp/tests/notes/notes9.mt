# note shorthand and transitions
1:1 / 2 \ 1 h 3:1 p 0 h 4:1
