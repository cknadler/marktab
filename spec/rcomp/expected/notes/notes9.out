e|-1-/-2-\-1-------------|
B|-----------------------|
G|-----------h-1-p-0-----|
D|-------------------h-1-|
A|-----------------------|
E|-----------------------|

